pkg/codegen/docs/gen.go

```diff
@@ -812,82 +748,26 @@ func (mod *modContext) genNestedTypes(member interface{}, resourceType bool) []d
 	// and if it appears in an input object and/or output object.
 	mod.getTypes(member, tokens)
 
-	isK8s := isKubernetesPackage(mod.pkg)
-
 	var typs []docNestedType
-	for token, tyUsage := range tokens {
+	for token := range tokens {
 		for _, t := range mod.pkg.Types {
 			switch typ := t.(type) {
 			case *schema.ObjectType:
 				if typ.Token != token || len(typ.Properties) == 0 {
 					continue
 				}
 
-				// Create maps to hold the per-language properties of this object and links to
-				// the API doc for each language.
+				// Create a map to hold the per-language properties of this object.
 				props := make(map[string][]property)
-				apiDocLinks := make(map[string]apiTypeDocLinks)
 				for _, lang := range supportedLanguages {
-					// The nested type may be under a different package in a language.
```

Comment:
All or most of this deletion was related to generating the per-language API doc link for each object type.